### ISSUE TITLE
Window doesn't resize to fit the content as expected

### ISSUE THREAD

**Issue by DmitriyYukhanov** [AvaloniaCommunity/MessageBox.Avalonia#104]
Hey,

I'm using v2.0.0 from the NuGet with Avalonia 0.10.13 on Win 10 and have a such problem: windows don't resize vertically to fit all the text without scroll:

![image](https://user-images.githubusercontent.com/883587/161229415-c50832da-cceb-4043-9649-2dae860e3ca9.png)

Here is how I call it:

```
private static readonly MessageBoxStandardParams ErrorMessageParams = new()
{
	ButtonDefinitions = ButtonEnum.Ok,
	ContentHeader = "Error",
	SystemDecorations = SystemDecorations.None,
	ShowInCenter = true,
	WindowStartupLocation = WindowStartupLocation.CenterOwner,
	Width = 400,
	MaxHeight = 1000,
	SizeToContent = SizeToContent.Height
};

// ...

ErrorMessageParams.ContentMessage = "Some long multiline text";

var messageBoxStandardWindow = MessageBoxManager.GetMessageBoxStandardWindow(ErrorMessageParams);
messageBoxStandardWindow.Show();
```

I expected it to resize height up to the MaxHeight, but it doesn't for some reason.

P.S. setting Height explicitly to 1000 doesn't change the actual MessageBox height either.

**Comment by DmitriyYukhanov** [comment]
And this is what happens when I try to resize the window manually with `CanResize = True;`

![gif](https://user-images.githubusercontent.com/883587/161231135-f7a68e53-9fc5-4f14-ba4c-be8e66de0529.gif)

**Comment by DmitriyYukhanov** [comment]
Seems to be reverse of the https://github.com/AvaloniaCommunity/MessageBox.Avalonia/issues/63 😅

**Comment by CreateLab** [comment]
Should be ok, check

**Comment by DmitriyYukhanov** [comment]
Thank you, I do confirm it's fixed for me now at the 2.0.1.
